src/app/view/perspectives/chart/visualizer/plot-maker/axis-draggable-plot-maker.ts

```diff
@@ -43,7 +45,30 @@ export abstract class AxisDraggablePlotMaker extends DraggablePlotMaker {
         },
       };
     }
-    return {};
+    return {
+      yaxis: {
+        range: this.createRange(),
+      },
+    };
+  }
+
+  private createRange(): any[] {
+    if (
+      !this.areBothYAxisPresented() ||
+      this.isAxisCategory(ChartAxisType.Y1) ||
+      this.isAxisCategory(ChartAxisType.Y2)
+    ) {
+      return null;
+    }
+
+    const values = this.chartData.sets.reduce((allValues, set) => {
+      const setValues = set.points
+        .filter(point => isNotNullOrUndefind(point.y) && isNumeric(point.y))
+        .map(point => point.y);
+      return [...allValues, ...setValues];
```

Comment:
I once hit a situation where values duplicated in an array created like that. For that purpose I added `uniqueArray` function. Isn't it needed here as well? Just trying to sell my luxurious stuff (the function)...

Reply:
It's not necessary to use it here :)

Reply:
It's not necessary to use it here :)